notebooks/book1/03/gauss_infer_2d.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    %pip install git+git://github.com/blackjax-devs/blackjax.git
</pre><p>This module is needed? We can remove not used modules. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>